src/shared/actions/polling.js

```diff
@@ -469,7 +462,7 @@ export const getAccountInfoForAllAccounts = (accountNames, notificationFn, quoru
  *
```

Comment:
It won't let me comment on the line above this one, but I think the documentation should be updated to reflect the added parameter

Reply:
@rajivshah3 https://github.com/iotaledger/trinity-wallet/pull/1779/commits/860fd5ba56dd598004d4d9d305f819d8ee57e914